IntegrationTests/SPMIntegration/fastlane/Scanfile

```diff
@@ -0,0 +1 @@
+catalyst_platform('macos')
```

Comment:
same here, temporary for testing but we should probably not keep it, we should test both iOS and macOS